packages/cli/src/config/schema.ts

```diff
@@ -470,11 +470,23 @@ export const schema = {
 		env: 'N8N_SSL_CERT',
 		doc: 'SSL Cert for HTTPS Protocol',
 	},
+	baseUrl: {
+		format: String,
+		default: '',
+		env: 'N8N_BASE_URL',
```

Comment:
added a new env variable, as `WEBHOOK_URL` was never meant to be used for non webhook urls (even though in some please it incorrectly is used that way).